base16-manager

```diff
@@ -64,6 +64,35 @@ check_dir() {
   fi
 }
 
+# move old config to new data directory or create new one
+# if it doesn't already exists
+check_data_dir() {
+  local option=$1
+
+  if ! dir_exists "$DATA_PATH"; then
+    local OLD_CONFIG_PATH="$HOME/.base16-manager"
+
+    # old dir exists - we need to move it to new path
+    if dir_exists "$OLD_CONFIG_PATH"; then
+      mv "$OLD_CONFIG_PATH" "$DATA_PATH"
+      echo
+      echo "base16-manager data was moved to new location $DATA_PATH"
+      echo
+
+      # user is not setting theme, we have to warn him and set something meantime
+      if [[ ! "$option" = "set" ]]; then
+        set_theme materia
+        echo
+        echo "Temporary set theme to materia, please choose your theme again with:"
+        echo "    base16-manager set yourfavoritetheme"
+        echo
+      fi
```

Comment:
Why is it necessary to reset the themes right now? Could you explain it please?

Reply:
I think this is to prevent apps from giving an error after moving the data directory. Maybe it's a better idea to start storing the set theme in the future.